.github/workflows/tidy.yml

```diff
@@ -0,0 +1,20 @@
+name: Tidy document
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - gh-pages
+
+jobs:
+  tidy:
+    name: Tidy up
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: brew install tidy-html5
+      - run: tidy -config tidyconfig.txt -o index.html index.html
+      - uses: peter-evans/create-pull-request@v3
```

Comment:
Is this what we typically do? Would it make sense to fork that action somewhere "official"?

Reply:
Yeah, that's typical. Open source, y'all. We don't share anything particularly sensitive here, so it's all good.